images/reviewer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:20.04
```

Comment:
@illusional Should this derive from the [existing driver base image](https://github.com/populationgenomics/analysis-runner/blob/main/driver/Dockerfile.base) or is there a better way to install the required dependencies?

Reply:
Only if it's going to be used as a driver image in the analysis runner - @hopedisastro, are you able to clarify if you're just using this as a job within a larger workflow, or planning to use the `--image` parameter in the analysis-runner.

Reply:
I think I will be using this as a job in Hail batch, to visualise read pileups of certain loci.